### ISSUE TITLE
prevent direct push to master

### ISSUE THREAD

**Issue by styk-tv** [tronprotocol/wiki#10]
no one should be able to push directly to master branch. the workflow should be as follows:

- create a branch or a forked branch and publish your change
- create a pr from new branch against master branch
- this will eventually trigger intermediate verification in travis-ci (currently peer review)
- only after build confirmed and reviewed we agree to merge into master
- merge into master will trigger rebuild and republish by RTD engine

task: can we verify no one can push directly to master and only through a PR?

**Comment by ruchern** [comment]
At this moment, it appears that we do not have rights to push to master and only the people in the `tronprotocol` organisation can merge the PR.

**Comment by ruchern** [comment]
I'll go ahead and close this issue since master cannot be pushed directly.
